### PR TITLE
feat: claims staging model

### PR DESCRIPTION
**Pull request by 90awlui**
## Description & motivation
Adding a staging model for claims and docs to support it.
Note: `created` in the base table is null so there isn't a created timestamp for this model

## Validation of models:
- I checked that none of the columns in the model were totally null (although some are null for the foreign keys)

## Checklist:

- [x] My pull request represents one logical piece of work.
- [x] My commits are related to the pull request and look clean.
- [x] My SQL follows the [dbt Labs style guide](https://github.com/dbt-labs/corp/blob/master/dbt_style_guide.md).
- [x] I have materialized my models appropriately.
- [x] I have added appropriate tests and documentation to any new models.
- [ ] I have updated the README file.
- [ ] I have mask the PII data.

### CODE REVIEW

File: models/marts/staging/stg_claims.sql
Comment (cecile-cecile) [inline models/marts/staging/stg_claims.sql:1]: nitpick: trailing whitespace

File: models/marts/staging/_stg.md
Comment (cecile-cecile) [inline models/marts/staging/_stg.md:1]: nitpick: trailing whitespace
In VS Code: `shift+cmd+P` (Trim Trailing Whitespace)

**Review by cecile-cecile — COMMENTED** [review]
Overall looks good! Only trailing whitespace nitpicks

File: models/marts/staging/_stg.md
Comment (90awlui) [reply models/marts/staging/_stg.md:1]: ty, didn't know about this

**Review by cecile-cecile — APPROVED** [review]
LGTM 🎉

**Pull request merged**